scripts/globals/regimes.lua

```diff
@@ -1202,6 +1202,16 @@ tpz.regime.bookOnEventFinish = function(player, option, regimeType)
 
             player:showText(player, msgOffset)
             player:showText(player, msgOffset + 1)
+
+            -- Records of Eminence: Undertake a FoV Training Regime
+            if player:getEminenceProgress(3) and regimeType == tpz.regime.type.FIELDS then
+                npcUtil.completeRecord(player, 3, { sparks = 100, xp = 500 })
```

Comment:
This is such a clean API 👨‍🍳

Reply:
Thanks! And thanks for the review. Good points brought up, particularly about coding standards which I'm sure will come up more often as we increase compiler strictness and deal with additional GCC8/9 warnings.